.github/workflows/fmt.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.9
+          python-version: 3.11
```

Comment:
This may have other implications.

Here's my previous PR: https://github.com/panther-labs/panther_analysis_tool/pull/355

Reply:
That's great! I'll close this one 😄